prod-aws/account-platform/analytics.tf

```diff
@@ -0,0 +1,16 @@
+resource "kafka_topic" "address_lookup_analytics_v1" {
+  name               = "address.lookup.analytics.v1"
+  replication_factor = 3
+  partitions         = 15
+}
+
+resource "kafka_topic" "legacy_account_braze_events_compacted" {
```

Comment:
Just checked what are the compaction settings on the actual prod fro this topic because I remember I changed them (and I didn't reflect it in the kafka topic applier, apologies for that..)

They're different unfortunately: https://account-platform-kafka-console.prod.aws.uw.systems/topics/legacy.account.braze.events.compacted#configuration